lib/shared/config/cli-flags.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var yargs = require('yargs');
+var camelCase = require('lodash.camelcase');
+
+function mergeToCliFlags(cliFlags, config, cliOptions) {
+  var argv = yargs(process.argv.slice(2)).argv;
```

Comment:
Why are you using `yargs` again here? Don't we already have everything parsed in `index.js`?